remotespark/datawidgets/encoding.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2015  aggftw@gmail.com
+# Distributed under the terms of the Modified BSD License.
+
+
+class Encoding(object):
```

Comment:
It's true they don't have anything to do with Altair at the moment, but Altair is using years of research that come up with its programming model, as well as the terminology for it.

I'm not aligning with Altair so much as what the visualization community has learned from the research Altair is built on. Sorry I wasn't clear.